content/courses/Ada_For_The_C_Embedded_Developer/chapters/07_Reusability.rst

```diff
@@ -914,8 +914,12 @@ both dynamic libraries together with the main application.
 You can find more details and special setting for library projects in the
 `GPRbuild documentation <https://docs.adacore.com/gprbuild-docs/html/gprbuild_ug/gnat_project_manager.html#library-projects>`_.
 
-.. todo::
-
-    Add link to section of *Embedded Ada* course that presents more advanced
-    features, as the one described in GNAT examples directory
-    (`share\examples\gnat\plugins`).
+.. admonition:: In the GNAT toolchain
+
+    The GNAT toolchain includes a more advanced example focusing on how to load
+    dynamic libraries at runtime. You can find it in the
+    :file:`share/examples/gnat/plugins` directory of the GNAT toolchain
+    installation. As described in the README file from that directory, this
+    example "comprises a main program which probes regularly for the existence
+    of shared libraries in known location. If such libraries are present, it
```

Comment:
in a known location

Reply:
Thanks for catching it! This mistake is in the original text as well...